seek.go

```diff
@@ -7,13 +7,12 @@ import (
 const errSeekBeforeBuffer Error = "cannot seek past the start of the buffer"
 const errSeekBeyondBuffer Error = "cannot seek past the end of the buffer"
 
-// NewSeekBuffer returns a buffer that allows streaming from any point in its saved data.
-// SeekBuffers are thread-safe.
```

Comment:
Why are these comments gone?

Reply:
Moved some of them to document the actual struct